### PR TITLE
Get TypeValidationError to also report the actual type passed.

### PR DESCRIPTION
**Pull request by danballance**
I was getting a type error that didn't make sense on AWS ("TypeValidationError: Passed value must be valid array type.") I'm fairly sure my request body is an array, however I can't tell much about what has actually been received. This PR adds some extra info into the error message so we can see the actual type received, as well as the expected.

### CODE REVIEW

File: tests/json_schema/test_type_validators.py [hunk 1]
```diff
@@ -100,5 +116,9 @@ def test_pass_validate_string(value: Any) -> None:
 
 @pytest.mark.parametrize("value", [True, False, [0], 2])
 def test_fail_validate_string(value: Any) -> None:
-    with pytest.raises(TypeValidationError):
+    with pytest.raises(TypeValidationError) as e:
         validate_string(value)
+    assert e.value.args[0] == (
+        "Passed value must be valid <class 'str'> type. "
+        f"Actual type passed was {type(value)}"
+    )
```
Comment (dkraczkowski) [inline tests/json_schema/test_type_validators.py:124]:
```suggestion
    )

```

Comment (szymonmiks-netguru) [inline tests/json_schema/test_type_validators.py:124]: No new line at the end of the file ;)

**Review by szymonmiks-netguru — APPROVED** [review]

**Review by dkraczkowski — CHANGES_REQUESTED** [review]

**Comment by dkraczkowski** [comment]
@danballance if possible could you also please update version inside `pyproject.toml` file and `__version__.py` file?

**Comment by danballance** [comment]
@dkraczkowski re the version numbers, I did consider this, but then I wasn't sure if there might be other fixes in progress and if I might clash. but I guess taking a best guess that you can correct is better than not adding one. So I'll do this for next time :)

**Pull request merged**